deploy/openshift/quay-recovery-endpoint.yaml

```diff
@@ -362,6 +365,12 @@ parameters:
   - name: QUAY_APP_READINESS_PROBE_TIMEOUT_SECONDS
     value: "10"
     displayName: quay app readiness probe timeout
+  - name: QUAY_DISABLE_MIN_REPLICAS_CHECK
+    value: "check_not_disabled"
```

Comment:
nit: should we set the default to `min-check-not-disabled` similar to the anti-affinity? that way it's clear what purpose the annotation serves if we don't provide a value for our deployment